staging/godeps-json-updater.go

```diff
@@ -43,12 +43,13 @@ type Godeps struct {
 	Deps         []Dependency
 }
 
-// rewrites the Godeps.ImportPath, removes the Deps whose ImportPath contains "k8s.io/kubernetes"
+// rewrites the Godeps.ImportPath, removes the Deps whose ImportPath contains "k8s.io/kubernetes" or "k8s.io/apimachinery".
```

Comment:
Is this from a different change? Looks like changes to this file should be reverted.